tutorial/consuming_packages/conanfile_py/complete_conanfile.py

```diff
@@ -25,5 +25,8 @@ def layout(self):
         multi = True if self.settings.get_safe("compiler") == "msvc" else False
         if multi:
             self.folders.generators = os.path.join("build", "generators")
+            self.folders.build = "build"
```

Comment:
For reference the whole can be simplified to:
```python
self.folders.build = "build" if multi else os.path.join("build", str(self.settings.build_type))
self.folders.generators = os.path.join(self.folders.build, "generators")
```
I am not saying to change it, it is more "classic" and educational as-is, better for non-python experts. Just saying how it would be in production code.